app/controllers/api/v1/users_controller.rb

```diff
@@ -1,17 +1,18 @@
 module Api
-    module V1
-        class UsersController < BaseController
-            def create
-                render :json => 'not yet implemented'.to_json, status: 501
-            end
+  module V1
+    class UsersController < BaseController
 
-            def index
-                render :json => 'not yet implemented'.to_json, status: 501
-            end
-
-            def destroy
-                render :json => 'not yet implemented'.to_json, status: 501
-            end
-        end
+      def create
+        render :json => 'not yet implemented'.to_json, status: 501
```

Comment:
same comment: more descriptive message. same for all actions in controller and remaining ones...